libbeat/docs/processors-using.asciidoc

```diff
@@ -748,3 +749,33 @@ The fields added to the event are looking as following:
 -------------------------------------------------------------------------------
 
 NOTE: The host information is refreshed every 5 minutes.
+
+[[dissect]]
+=== Dissect
+
+The `dissect` processor allows you to use the `dissect` syntax to tokenizer incoming string into
+defined patterns.
+
+
+Note that if the key exists in the `target_field`, the original value will be replaced.
+
+TODO: need to finish
+
+[source,yaml]
+-------
+processors:
+- dissect:
+    tokenizer: "%{key1} %{key2}"
+    field: "message"
+    target_field: "extracted"
+-------
+
+The `dissect` processor has the following configuration settings:
+
+`field`:: (Optional) The event field to tokenize. Default is `message`.
+
+`target_field`:: (Optional) The name of the field where the values will be extracted. When an empty
```

Comment:
It's more the target object or `prefix`? If I understand this correctly the data will be under `extracted.key1, extracted.key2`?